xbmc/platform/linux/input/LIRC.cpp

```diff
@@ -61,7 +73,12 @@ CRemoteControl::~CRemoteControl()
     fclose(m_file);
 }
 
-void CRemoteControl::SetEnabled(bool value)
+std::string CRemoteControl::GetMapFile()
+{
+  return LIRC_MAP_FILENAME;
+}
+
+void CRemoteControl::SetEnabled(bool bEnabled)
```

Comment:
`bool value` or alue has to be replaced with `bEnabled` in the whole function

Reply:
oops. thanks.